src/DL/TransactionHistoryHandler.java

```diff
@@ -0,0 +1,77 @@
+package DL;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
+public class TransactionHistoryHandler {
+
+    private final String path;
+
+
+    public TransactionHistoryHandler(String path) {
+        create(path);
+        this.path = path;
+    }
+
+
+    private boolean create(String path) {
```

Comment:
no need to return boolean (or something else) if you don't plan to use result of this method anywhere

Reply:
private void create(String path) {
        try {
            this.file.createNewFile();
        } catch (IOException e) {
            System.out.println(e.getMessage());
        }
    }

OK? @ychepel

Reply:
Done